pkg/cmd/server/origin/ensure.go

```diff
@@ -54,15 +54,7 @@ func (c *MasterConfig) ensureOpenShiftInfraNamespace() {
 		return
 	}
 
-	roleAccessor := policy.NewClusterRoleBindingAccessor(c.ServiceAccountRoleBindingClient())
-	for _, saName := range bootstrappolicy.InfraSAs.GetServiceAccounts() {
-		_, err := c.KubeClientsetInternal().Core().ServiceAccounts(ns).Create(&kapi.ServiceAccount{ObjectMeta: metav1.ObjectMeta{Name: saName}})
-		if err != nil && !kapierror.IsAlreadyExists(err) {
-			glog.Errorf("Error creating service account %s/%s: %v", ns, saName, err)
-		}
-
-		role, _ := bootstrappolicy.InfraSAs.RoleFor(saName)
-
+	for _, role := range bootstrappolicy.ControllerRoles() {
```

Comment:
> doesn't this omit the kube bootstrappolicy.ControllerRoles()?

We should need any of this moving forward and those are particularly easy to chase.  We should stay strict in 3.7